custom_components/tuya_local/devices/airquality6in1.yaml

```diff
@@ -0,0 +1,70 @@
+name: Air quality monitor 6 in 1
+products:
+  - id: ugjbnqj2mffaexp5
+    name: Air Housekeeper
+primary_entity:
+  entity: sensor
+  name: PM2.5
+  class: volatile_organic_compounds
```

Comment:
class for a PM2.5 sensor should be pm25, and then it will not need a name, as the class defines a translated name already.